sandbox/grist/table.py

```diff
@@ -97,11 +100,14 @@ def lookupRecords(Field in lookup Table = value):
     """
     return self.table.lookup_records(**field_value_pairs)
 
-  def lookupOne(Field in lookup Table = value):
+  def lookupOne(self, **field_value_pairs):
     """
-    Returns a Record matching the given field=value arguments. The value may be expressed as a string 
-    between `“ ”` or a field in the current table (examples below). If multiple records match, returns 
-    one of them. If none match, returns the special empty record.
+    Name: lookupOne
+    Usage: UserTable.__lookupOne__(Field_In_Lookup_Table=value, ...)
+    Returns a Record matching the given field=value arguments. The value may be any expression, 
+    most commonly a field in the current row (e.g. $SomeField) or a constant (e.g. a quoted string 
```

Comment:
I suggest putting $SomeField and "Some Value" in backticks, so they look like code examples.